lib/taurus/core/taurusmanager.py

```diff
@@ -120,15 +125,28 @@ def addJob(self, job, callback=None, *args, **kw):
         :param args: (list) list of arguments passed to the job
         :param kw: (dict) keyword arguments passed to the job
         """
-        if self._serialization_mode == TaurusSerializationMode.Concurrent:
+        if kw.has_key("taurus_serialization_mode"):
```

Comment:
Do we need to support adding the serialization mode kwarg here? (the previous implementation did not require it...)
If we do, at least we should document it in the method docstring, because ` taurus_serialization_mode` is not passed to the job (which is what the docstring says)

Reply:
Yes, we need. The previous implementation treat the serial mode at the scheme level. So,  the scheme overwrite the setting at the scheme level and not at manager level

> If we do, at least we should document it in the method docstring, because taurus_serialization_mode is not passed to the job (which is what the docstring says)

Agree. I will